### PR TITLE
Fix run_setup_tests handling for destinations

### PR DESCRIPTION
**Pull request by beevital**

### CODE REVIEW

File: fivetran/resource_destination.go [hunk 1]
```diff
@@ -25,14 +25,11 @@ func resourceDestination() *schema.Resource {
...
-			"trust_fingerprints": {Type: schema.TypeBool, Optional: true}, // T-112419, ForceNew can be removed and the field can be updated
-			// "run_setup_tests" default value is true. It is set as required to avoid confusion, so the expected behaviour should
-			// be explicitly declared.
-			"run_setup_tests": {Type: schema.TypeBool, Required: true},
-			"setup_status":    {Type: schema.TypeString, Computed: true},
-			// "setup_tests": ... // missing /T-112419
-			"last_updated": {Type: schema.TypeString, Computed: true}, // internal
+			"trust_certificates": {Type: schema.TypeBool, Optional: true},
+			"trust_fingerprints": {Type: schema.TypeBool, Optional: true},
+			"run_setup_tests":    {Type: schema.TypeBool, Optional: true, Default: true},
```
Comment (akankaluga) [inline fivetran/resource_destination.go:30]: As I remember, we have setup tests turned off by default in connector. Should we have same defaults?

Comment (beevital) [reply fivetran/resource_destination.go:30]: I don't want to change the logic now. This PR keeps existing logic.

Further we will review defaults, and perform the same model for `run_setup_tests` field in connector resource.

Comment (beevital) [reply fivetran/resource_destination.go]: We have discussed it and decided to change logic. Previously it was mandatory to specify this parameter, so there was no default behavior. Let's make consistent with conenctors.

**Review by Fivetran-RomanKolesnikov — APPROVED** [review]
Looks good!

**Pull request merged**